# HG changeset patch
# User Vytautas Janƒçauskas <unaudio@gmail.com>
# Date 1379452588 -10800
#      Wed Sep 18 00:16:28 2013 +0300
# Node ID ee3ec3f0235861feedac108106dad9ee983d1161
# Parent  480407a8d22620851f09892f2f9a06f20b2a7a3e
maint: moved @audiorecorder and @audioplayer folders inside the audio folder

diff --git a/scripts/@audioplayer/__get_properties__.m b/scripts/audio/@audioplayer/__get_properties__.m
rename from scripts/@audioplayer/__get_properties__.m
rename to scripts/audio/@audioplayer/__get_properties__.m
diff --git a/scripts/@audioplayer/audioplayer.m b/scripts/audio/@audioplayer/audioplayer.m
rename from scripts/@audioplayer/audioplayer.m
rename to scripts/audio/@audioplayer/audioplayer.m
diff --git a/scripts/@audioplayer/display.m b/scripts/audio/@audioplayer/display.m
rename from scripts/@audioplayer/display.m
rename to scripts/audio/@audioplayer/display.m
diff --git a/scripts/@audioplayer/get.m b/scripts/audio/@audioplayer/get.m
rename from scripts/@audioplayer/get.m
rename to scripts/audio/@audioplayer/get.m
diff --git a/scripts/@audioplayer/isplaying.m b/scripts/audio/@audioplayer/isplaying.m
rename from scripts/@audioplayer/isplaying.m
rename to scripts/audio/@audioplayer/isplaying.m
diff --git a/scripts/@audioplayer/module.mk b/scripts/audio/@audioplayer/module.mk
rename from scripts/@audioplayer/module.mk
rename to scripts/audio/@audioplayer/module.mk
diff --git a/scripts/@audioplayer/pause.m b/scripts/audio/@audioplayer/pause.m
rename from scripts/@audioplayer/pause.m
rename to scripts/audio/@audioplayer/pause.m
diff --git a/scripts/@audioplayer/play.m b/scripts/audio/@audioplayer/play.m
rename from scripts/@audioplayer/play.m
rename to scripts/audio/@audioplayer/play.m
diff --git a/scripts/@audioplayer/playblocking.m b/scripts/audio/@audioplayer/playblocking.m
rename from scripts/@audioplayer/playblocking.m
rename to scripts/audio/@audioplayer/playblocking.m
diff --git a/scripts/@audioplayer/resume.m b/scripts/audio/@audioplayer/resume.m
rename from scripts/@audioplayer/resume.m
rename to scripts/audio/@audioplayer/resume.m
diff --git a/scripts/@audioplayer/set.m b/scripts/audio/@audioplayer/set.m
rename from scripts/@audioplayer/set.m
rename to scripts/audio/@audioplayer/set.m
diff --git a/scripts/@audioplayer/stop.m b/scripts/audio/@audioplayer/stop.m
rename from scripts/@audioplayer/stop.m
rename to scripts/audio/@audioplayer/stop.m
diff --git a/scripts/@audioplayer/subsasgn.m b/scripts/audio/@audioplayer/subsasgn.m
rename from scripts/@audioplayer/subsasgn.m
rename to scripts/audio/@audioplayer/subsasgn.m
diff --git a/scripts/@audioplayer/subsref.m b/scripts/audio/@audioplayer/subsref.m
rename from scripts/@audioplayer/subsref.m
rename to scripts/audio/@audioplayer/subsref.m
diff --git a/scripts/@audiorecorder/__get_properties__.m b/scripts/audio/@audiorecorder/__get_properties__.m
rename from scripts/@audiorecorder/__get_properties__.m
rename to scripts/audio/@audiorecorder/__get_properties__.m
diff --git a/scripts/@audiorecorder/audiorecorder.m b/scripts/audio/@audiorecorder/audiorecorder.m
rename from scripts/@audiorecorder/audiorecorder.m
rename to scripts/audio/@audiorecorder/audiorecorder.m
diff --git a/scripts/@audiorecorder/display.m b/scripts/audio/@audiorecorder/display.m
rename from scripts/@audiorecorder/display.m
rename to scripts/audio/@audiorecorder/display.m
diff --git a/scripts/@audiorecorder/get.m b/scripts/audio/@audiorecorder/get.m
rename from scripts/@audiorecorder/get.m
rename to scripts/audio/@audiorecorder/get.m
diff --git a/scripts/@audiorecorder/getaudiodata.m b/scripts/audio/@audiorecorder/getaudiodata.m
rename from scripts/@audiorecorder/getaudiodata.m
rename to scripts/audio/@audiorecorder/getaudiodata.m
diff --git a/scripts/@audiorecorder/getplayer.m b/scripts/audio/@audiorecorder/getplayer.m
rename from scripts/@audiorecorder/getplayer.m
rename to scripts/audio/@audiorecorder/getplayer.m
diff --git a/scripts/@audiorecorder/isrecording.m b/scripts/audio/@audiorecorder/isrecording.m
rename from scripts/@audiorecorder/isrecording.m
rename to scripts/audio/@audiorecorder/isrecording.m
diff --git a/scripts/@audiorecorder/module.mk b/scripts/audio/@audiorecorder/module.mk
rename from scripts/@audiorecorder/module.mk
rename to scripts/audio/@audiorecorder/module.mk
diff --git a/scripts/@audiorecorder/pause.m b/scripts/audio/@audiorecorder/pause.m
rename from scripts/@audiorecorder/pause.m
rename to scripts/audio/@audiorecorder/pause.m
diff --git a/scripts/@audiorecorder/play.m b/scripts/audio/@audiorecorder/play.m
rename from scripts/@audiorecorder/play.m
rename to scripts/audio/@audiorecorder/play.m
diff --git a/scripts/@audiorecorder/record.m b/scripts/audio/@audiorecorder/record.m
rename from scripts/@audiorecorder/record.m
rename to scripts/audio/@audiorecorder/record.m
diff --git a/scripts/@audiorecorder/recordblocking.m b/scripts/audio/@audiorecorder/recordblocking.m
rename from scripts/@audiorecorder/recordblocking.m
rename to scripts/audio/@audiorecorder/recordblocking.m
diff --git a/scripts/@audiorecorder/resume.m b/scripts/audio/@audiorecorder/resume.m
rename from scripts/@audiorecorder/resume.m
rename to scripts/audio/@audiorecorder/resume.m
diff --git a/scripts/@audiorecorder/set.m b/scripts/audio/@audiorecorder/set.m
rename from scripts/@audiorecorder/set.m
rename to scripts/audio/@audiorecorder/set.m
diff --git a/scripts/@audiorecorder/stop.m b/scripts/audio/@audiorecorder/stop.m
rename from scripts/@audiorecorder/stop.m
rename to scripts/audio/@audiorecorder/stop.m
diff --git a/scripts/@audiorecorder/subsasgn.m b/scripts/audio/@audiorecorder/subsasgn.m
rename from scripts/@audiorecorder/subsasgn.m
rename to scripts/audio/@audiorecorder/subsasgn.m
diff --git a/scripts/@audiorecorder/subsref.m b/scripts/audio/@audiorecorder/subsref.m
rename from scripts/@audiorecorder/subsref.m
rename to scripts/audio/@audiorecorder/subsref.m
diff --git a/scripts/audio/module.mk b/scripts/audio/module.mk
--- a/scripts/audio/module.mk
+++ b/scripts/audio/module.mk
@@ -4,15 +4,44 @@ audio_FCN_FILES = \
   audio/lin2mu.m \
   audio/loadaudio.m \
   audio/mu2lin.m \
   audio/playaudio.m \
   audio/record.m \
   audio/saveaudio.m \
   audio/setaudio.m \
   audio/wavread.m \
-  audio/wavwrite.m
+  audio/wavwrite.m \
+  audio/@audioplayer/__get_properties__.m \
+  audio/@audioplayer/audioplayer.m \
+  audio/@audioplayer/display.m  \
+  audio/@audioplayer/get.m  \
+  audio/@audioplayer/isplaying.m  \
+  audio/@audioplayer/pause.m  \
+  audio/@audioplayer/play.m  \
+  audio/@audioplayer/playblocking.m  \
+  audio/@audioplayer/resume.m  \
+  audio/@audioplayer/set.m \
+  audio/@audioplayer/stop.m \
+  audio/@audioplayer/subsasgn.m \
+  audio/@audioplayer/subsref.m \
+  audio/@audiorecorder/__get_properties__.m \
+  audio/@audiorecorder/audiorecorder.m \
+  audio/@audiorecorder/display.m \
+  audio/@audiorecorder/get.m \
+  audio/@audiorecorder/getaudiodata.m \
+  audio/@audiorecorder/getplayer.m \
+  audio/@audiorecorder/isrecording.m \
+  audio/@audiorecorder/pause.m \
+  audio/@audiorecorder/play.m \
+  audio/@audiorecorder/record.m \
+  audio/@audiorecorder/recordblocking.m \
+  audio/@audiorecorder/resume.m \
+  audio/@audiorecorder/set.m \
+  audio/@audiorecorder/stop.m  \
+  audio/@audiorecorder/subsasgn.m  \
+  audio/@audiorecorder/susbref.m
 
 FCN_FILES += $(audio_FCN_FILES)
 
 PKG_ADD_FILES += audio/PKG_ADD
 
 DIRSTAMP_FILES += audio/$(octave_dirstamp)
